utils/check_code_style.py

```diff
@@ -239,13 +239,15 @@ def sanitize(lines, skip_checks=False):
 					i += 1
 					start_index = i + 1
 				continue
-			if (not is_string) and first_two == "//":
+			commentMatch = re.search("^//(/<?)?", line[i:i + 4])
```

Comment:
It would probably be worth precompiling this regex (around line 106). Should be something like
`singleLineComment = re.compile("^//(/<?)?")`, and you should just be able to do `re.search(singleLineComment, line[i:i + 4]` with that.

Reply:
Yes, that is nice.